src/browser/components/LoginModal.jsx

```diff
@@ -54,7 +53,10 @@ export default class LoginModal extends React.Component {
                 <FormControl
                   type='text'
                   placeholder='User Name'
-                  ref={this.usernameRef}
+                  value={this.state.username}
```

Comment:
the FormControl does not have to be re-rendered every time there is a key stroke

Reply:
Thanks, removed